addon/components/frost-action-bar.js

```diff
@@ -67,38 +86,131 @@ export default Component.extend({
 
   // == Computed Properties ===================================================
 
-  @readOnly
-  @computed('selectedItems.[]')
-  isVisible (selectedItems) {
-    return !isEmpty(selectedItems)
-  },
-
   @readOnly
   @computed('selectedItems.[]')
   selectedItemsLabel (selectedItems) {
     return this.i18n.formatSelectedItemsLabel(selectedItems.length)
   },
 
   @readOnly
-  @computed('controls')
-  isControlsObject (controls) {
-    return typeOf(controls) === 'object'
+  @computed('controls', 'selectedItems.[]', 'moreActionButtons')
+  /**
+   * gets controls for the action bar
+   * @param {array|object} controls - array or hash of controls
+   * @param {array} selectedItems - array of selected items for filtering controls
+   * @param {boolean|object|array} moreActionButtons - array or hash of extra controls
+   *  or flag indicating whether to generate them automaticaly
```

Comment:
change to `automatically`

Reply:
ty

Reply:
Also, fixed the spellchecker for my comments...